client/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "coursemology",
+  "version": "2.0.0",
+  "description": "Coursemology Frontend",
+  "engines": {
+    "node": "5.10.0",
+    "npm": "3.5.0"
+  },
+  "scripts": {
+    "build:test": "webpack --config webpack.config.js",
+    "build:production": "NODE_ENV=production webpack --config webpack.config.js",
+    "build:development": "webpack -w --config webpack.config.js",
+    "lint": "eslint . --ext .js --ext .jsx --cache",
+    "lint-fix": "eslint . --ext .js --ext .jsx --cache --fix"
+  },
+  "cacheDirectories": [
+    "node_modules",
+    "client/node_modules"
+  ],
+  "dependencies": {
+    "babel": "^6.5.2",
+    "babel-cli": "^6.6.5",
+    "babel-core": "^6.7.4",
+    "babel-loader": "^6.2.4",
+    "babel-polyfill": "^6.7.4",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-react": "^6.5.0",
+    "babel-preset-stage-0": "^6.5.0",
+    "babel-runtime": "^6.6.1",
+    "css-loader": "^0.25.0",
+    "es5-shim": "^4.5.7",
+    "expose-loader": "^0.7.1",
+    "immutable": "^3.8.1",
+    "imports-loader": "^0.6.5",
+    "mirror-creator": "1.1.0",
+    "node-sass": "^3.10.1",
+    "react": "^0.14.8 || ^15.0.0",
+    "react-bootstrap": "^0.30.5",
+    "react-css-modules": "^3.7.10",
```

Comment:
Yeah, I think we can remove it.

Main considerations:
- Namespace clashes are unlikely since the styles are included on a per page basis. Even if have, shouldn't be hard to manage.
- One useful benefit of react-css-modules is that it flags a warning if the css module is absent. We don't need this since the linter will already complain.